src/Dialog/index.stories.js

```diff
@@ -160,21 +156,42 @@ export const PopoverDialog = () => {
         }
         `}
       </style>
-      <Popover closeOnContentClick content={
-        <div className="popover-content">
-          <div tabIndex="0" role="button" onClick={() => { setIsDialogOpen(true) }} onKeyDown={() => { }}>
-            Open Modal
-          </div>
-          <div tabIndex="0" role="button" onClick={() => { }} onKeyDown={() => { }}>
-            Does Nothing
+      <Popover
```

Comment:
This might be a separate issue, but when I try to close the popover open modal by clicking 'x', the modal does not close.

Reply:
filing a follow up to investigate